prisma/schema.prisma

```diff
@@ -159,25 +159,30 @@ model Image {
 // this is for validator service
 model CandidUserInfo {
   id               Int     @id @default(autoincrement())
+  createdAt       DateTime @default(now())
+  updatedAt       DateTime @updatedAt
+
   claimId          Int?    @unique
   firstName        String?
   lastName         String?
   candid_entity_id String
   email            String
   profileURL       String
+  response         ResponseStatus?
 }
 
 model ValidationRequest {
-  id        Int      @id @default(autoincrement())
-  createdAt DateTime @default(now())
+  id                Int              @id @default(autoincrement())
+  createdAt         DateTime         @default(now())
+  updatedAt         DateTime         @updatedAt
 
   context           String
   validatorName     String
   validatorEmail    String
-  claimId           Int              // the id of the candid user claim (the root claim)
-  validationClaimId Int?             @unique // the id of validation claim
+  claimId           Int
+  validationClaimId Int?             @unique
   validationStatus  ValidationStatus @default(PENDING)
-  response          ResponseStatus?
```

Comment:
i think response should be with the request, NOT the user, i woudl keep it here

Reply:
hmm that thats depend on where we return the response is it for the candid user or for each request?
in the proposal doc we return the colors on the claimant (the candid user I guess)